scripts/vagrant/provision/manifests/m3coordinator-two.yaml

```diff
@@ -78,6 +78,25 @@ spec:
     app: m3coordinator
   type: ClusterIP
 
+---
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: m3coordinator
+  name: m3coordinator-dedicated-test-cluster-node-local
+  namespace: default
+spec:
+  ports:
+  - name: coordinator
+    port: 7201
+    protocol: TCP
+    targetPort: 7201
+    nodePort: 30000
+  selector:
+    app: m3coordinator
+  type: NodePort
```

Comment:
More of a question, what does this do as opposed to the definition for `m3coordinator-dedicated-test-cluster` above?

Reply:
i'm exposing the m3coordinator port locally on the node.